src/components/TableFilter.js

```diff
@@ -264,6 +268,24 @@ class TableFilter extends React.Component {
     );
   }
 
+  renderCustomField(column, index) {
+    const { classes, filterList, options } = this.props;
+    const display =
+      (column.filterOptions && column.filterOptions.display) ||
+      (options.filterOptions && options.filterOptions.display);
+
+    return (
+      <GridListTile key={index} cols={1}>
+        <div className={classes.textFieldRoot}>
+          <FormControl key={index}>
+            {(display && display(filterList, this.handleCustomChange, index, column)) ||
+              "Property 'display' is required when using custom filter type."}
```

Comment:
Oops, sorry, one more thing. What I was actually thinking here was to not have anything display, and instead show a warning to this effect in the console.

My reasoning is that I don't want the UI to be given a default means of displaying errors/warning information, as I think that should be up to the implementation.

Reply:
Oh, my fault for misunderstanding that. Pushed changes to just be console log and show nothing when display isn't passed in